### PR TITLE
[COMDLG32] Fix ico3 not shown in Print dialog

### PR DESCRIPTION
**Pull request by zecarlos1957**
[COMDLG32] Fix ico3 not shown

We need to update the window by calling SetWindowPos () in the STATIC_SetIcon (...) function in <dll / win32 / comctl32 / static.c>

Small adjustment in the position of the icon

### CODE REVIEW

**Comment by zecarlos1957** [comment]
prtAfter

**Comment by zecarlos1957** [comment]
I am not sure if this is the best solution.

File: dll/win32/comdlg32/lang/cdlg_Cs.rc [hunk 1]
```diff
@@ -369,7 +369,7 @@ FONT 8, "MS Shell Dlg"
 
     GROUPBOX        "Kopie",         grp2, 160, 92, 120,64, WS_GROUP
     LTEXT           "Počet &kopií:",stc5,168,105,68,8
-    ICON            "",               ico3, 170,131,  76,24, WS_GROUP | SS_CENTERIMAGE
+    ICON            "",               ico3, 210,131,  76,24, WS_GROUP//.| SS_CENTERIMAGE
```
Comment (SergeGautherie) [inline dll/win32/comdlg32/lang/cdlg_Cs.rc:372]:
```suggestion
    ICON            "",               ico3, 210,131,  76,24, WS_GROUP// | SS_CENTERIMAGE
```

File: dll/win32/comdlg32/lang/cdlg_Da.rc [hunk 2]
```diff
@@ -369,7 +369,7 @@ FONT 8, "MS Shell Dlg"
 
     GROUPBOX        "Kopier",         grp2, 160, 92, 120,64, WS_GROUP
     LTEXT           "Antal &kopier:",stc5,168,105,68,8
-    ICON            "",               ico3, 170,131,  76,24, WS_GROUP | SS_CENTERIMAGE
+    ICON            "",               ico3, 210,131,  76,24, WS_GROUP//.| SS_CENTERIMAGE
```
Comment (SergeGautherie) [inline dll/win32/comdlg32/lang/cdlg_Da.rc:372]:
```suggestion
    ICON            "",               ico3, 210,131,  76,24, WS_GROUP// | SS_CENTERIMAGE
```

File: dll/win32/comdlg32/lang/cdlg_Pt.rc [hunk 3]
```diff
@@ -353,7 +353,7 @@ FONT 8, "MS Shell Dlg"
     LTEXT "", stc13, 65, 72, 212, 10, SS_NOPREFIX | SS_LEFTNOWORDWRAP
     GROUPBOX "Cópias", grp2, 160, 92, 120, 64, WS_GROUP
     LTEXT "Número de &cópias:", stc5, 168, 105, 68, 8
-    ICON "", ico3, 170, 131, 76, 24, WS_GROUP | SS_CENTERIMAGE
+    ICON "", ico3, 210, 131, 76, 24, WS_GROUP //| SS_CENTERIMAGE
```
Comment (SergeGautherie) [inline dll/win32/comdlg32/lang/cdlg_Pt.rc:356]:
```suggestion
    ICON "", ico3, 210, 131, 76, 24, WS_GROUP // | SS_CENTERIMAGE
```

File: dll/win32/comdlg32/lang/cdlg_Bg.rc [hunk 4]
```diff
@@ -369,7 +369,7 @@ FONT 8, "MS Shell Dlg"
 
     GROUPBOX        "Копия",         grp2, 160, 92, 120,64, WS_GROUP
     LTEXT           "Брой &копия:",stc5,168,105,68,8
-    ICON            "",               ico3, 170,131,  76,24, WS_GROUP | SS_CENTERIMAGE
+    ICON            "",               ico3, 210,131,  76,24, WS_GROUP// | SS_CENTERIMAGE
```
Comment (ThFabba) [inline dll/win32/comdlg32/lang/cdlg_Bg.rc:372]: If `SS_CENTERIMAGE` is wrong, please remove it. If it's correct but there's a bug in ROS that means we can't use it, that deserves a comment.

Comment (zecarlos1957) [reply dll/win32/comdlg32/lang/cdlg_Bg.rc]: We definitely can't use SS_CENTERIMAGE. I made a test program for windows and it has exactly the same behavior; when using SS_CENTERIMAGE the icon is not shown.

**Comment by zecarlos1957** [comment]
I need help to create and send this patch to WINE

**Review by katahiromz — APPROVED** [review]

File: dll/win32/comdlg32/rsrc.rc [hunk 5]
```diff
@@ -146,9 +146,6 @@
 #ifdef LANGUAGE_UK_UA
     #include "lang/cdlg_Uk.rc"
 #endif
-#ifdef LANGUAGE_ZH_TW
-    #include "lang/cdlg_Tw.rc"
-#endif
```
Comment (HBelusca) [inline dll/win32/comdlg32/rsrc.rc:151]: Why is this line (and associated file) removed?

**Review by HBelusca — COMMENTED** [review]
You may as well remove the following comment (x2) in dll/win32/comdlg32/printdlg.c:
`FIXME: The ico3 is not displayed for some reason. I don't know why`

File: dll/win32/comdlg32/rsrc.rc (hunk 5 above)
Comment (binarymaster) [reply dll/win32/comdlg32/rsrc.rc]: It was a wrong way to perform rebase. Fixed by rebasing it properly.

**Pull request merged**